client/src-worker/turbo-resolver/registries/npm-http.js

```diff
@@ -13,10 +13,16 @@ export class NpmHttpRegistry {
     }
 
     if (!this.fetching[name]) {
-      this.fetching[name] = fetch(`${this.registryUrl}/${name}`).then(response => {
+      this.fetching[name] = fetch(`${this.registryUrl}/${name}`).then(async response => {
         if(!response.ok){
-          console.error(`Could not load npm registry for ${name}: ${response.statusText}`);
-          throw new Error(response.statusText);
+          // npm can send a json error
```

Comment:
Could you provide me a URL to generate a OK response with json error in body? This sounds weird npm behavior to report error on 20x response.